src/language-server.ts

```diff
@@ -71,9 +71,10 @@ function log(...values: unknown[]) {
     connection.console.log(message);
   }
 }
-if (false as boolean) {
-  log(`Log is useful when developing, even if we don't use it currently.`);
-}
+
+// So that we can just console.log and console.error as usual.
```

Comment:
Does this mean that if this module gets imported, all `console.log` behavior changes for the whole process?

Is it possible to isolate that more cleanly so that there isn't a global side-effect?

Reply:
This module is a binary. If it's imported, it will crash immediately unless it's been launched with a node RPC channel.

We could separate it out into a binary script and a library with the implementation if you think that'd be cleaner.

Reply:
Ah I see. Seems ok.

Maybe update the top of this file description to give a little more context? It's currently documented as though it were inside the extension folder, so that could help in the future.